include/nvector/nvector_kokkos.hpp

```diff
@@ -567,9 +567,9 @@ class Vector : public sundials::impl::BaseNVector,
 
   operator N_Vector() const override { return object_.get(); }
 
-  N_Vector Convert() override { return object_.get(); }
+  N_Vector get() override { return object_.get(); }
 
-  N_Vector Convert() const override { return object_.get(); }
+  N_Vector get() const override { return object_.get(); }
```

Comment:
Breaking change? Why the name change?

Reply:
Explained elsewhere, but basically this function will be typed many  times by Python users so a shorter name is needed, also we decided on this many months ago.